easystacks/software.eessi.io/2023.06/icelake_cclake/001-eb-4.9.4.yml

```diff
@@ -0,0 +1,4 @@
+easyconfigs:
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/EasyBuild/4.8.2/easybuild/reprod/EasyBuild-4.8.2.eb:
```

Comment:
I didn't follow the whole discussion, but why does this need to be the full path to an already installed easyconfig?

What was the problem with specifying `EasyBuild-4.8.2.eb` here (which is included in EasyBuild 4.9.4)?

Reply:
We're using a script that generates easystacks based on an existing stack. In this case, we're basically cloning the haswell stack by using the easyconfigs+easyblocks (+EB version) found in the installation dirs of all apps.